app/src/main/res/values/userpreferences.xml

```diff
@@ -32,6 +32,9 @@
   <string name="userprefs_show_colored_comments_tree_on">Lines are colored</string>
   <string name="userprefs_show_colored_comments_tree_off">Same color for all lines</string>
 
+  <string name="userprefs_show_color_replication_icons">Color-blind friendly mode</string>
+  <string name="userprefs_show_color_replication_icons_summary">Replicate any significant color with icons</string>
```

Comment:
Not sure if this is the best way to explain what does it actually do in one sentence